addons/mail_bot_hr/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "mail_bot_hr",
+    'summary': """Add OdooBot state and notifications in user form.""",
+    'description': "",
+    'website': "https://www.odoo.com/page/discuss",
+    'category': 'Productivity/Discuss',
+    'version': '1.0',
+    'depends': ['base', 'mail'],
+    'application': False,
+    'installable': True,
+    'auto_install': False,
```

Comment:
I would make it auto install, as people will probably not manually search for it.